infrastructure/helm/quantumserverless/values.yaml

```
@@ -17,6 +17,7 @@ gatewayEnable: true
 gateway:
   image:
     pullPolicy: Always
```

not related to this PR, but for consistency with the others...

```suggestion
    pullPolicy: IfNotPresent
```